tutorials/k8s-kapsule-multi-az/index.mdx

```diff
@@ -0,0 +1,614 @@
+Then, disable the default Kubernetes service creation by the Helm chart as it we will be created through Terraform later on to have multiple `LoadBalancers`.
+Finally, add a short snippet to the Nginx configuration to expose a `/up` path, which will be used in your DNS configuration later.
+    ```
+    resource "helm_release" "nginx_ingress" {
+        name      = "ingress-nginx"
+        namespace = "ingress-nginx"
+
+        create_namespace = true
+
+        repository = "https://kubernetes.github.io/ingress-nginx"
+        chart      = "ingress-nginx"
+
+        values = [
+            <<-EOT
+            controller:
+              replicaCount: 6
+
+              topologySpreadConstraints:
+                - topologyKey: topology.kubernetes.io/zone
+                  maxSkew: 1
+                  whenUnsatisfiable: DoNotSchedule
+                  labelSelector:
+                    matchLabels:
+                      app.kubernetes.io/name: ingress-nginx
+                      app.kubernetes.io/instance: ingress-nginx
+                      app.kubernetes.io/component: controller
+              affinity:
+                podAntiAffinity:
+                  requiredDuringSchedulingIgnoredDuringExecution:
+                  - labelSelector:
+                      matchLabels:
+                        app.kubernetes.io/name: ingress-nginx
+                        app.kubernetes.io/instance: ingress-nginx
+                        app.kubernetes.io/component: controller
+                    topologyKey: "kubernetes.io/hostname"
+              service:
+                enabled: false
+              config:
+                location-snippet: |
+                  location /up {
+                    return 200 'up';
+                  }
+            EOT
+        ]
+    }
+    ```
+
+4. Now add Kubernetes services of type `LoadBalancer` to expose your `ingress-nginx` pods. The important part here is that we will use the `service.beta.kubernetes.io/scw-loadbalancer-zone` annotation to create a load balancer per AZ. This annotation is used by the `scaleway-cloud-controller-manager` to create a load balancer in a specific AZ. In the `fr-par` region, load balancers are available in two AZs, so we will have two load balancers. Each load balancer dispatches traffic to the healthy nodes of all the AZs. This means, in case of the loss of one AZ where one of the two load balancers is located, the other load balancer will still be available and will dispatch traffic to the healthy nodes, and their healthy `ingress-nginx` pods, on the other AZs.
```

Comment:
```suggestion
4. Add Kubernetes services of type `LoadBalancer` to expose your `ingress-nginx` pods. The important part here is that we will use the `service.beta.kubernetes.io/scw-loadbalancer-zone` annotation to create one load balancer per AZ. This annotation is used by the `scaleway-cloud-controller-manager` to create a load balancer in a specific AZ. In the `fr-par` region, load balancers are available in two AZs, so we will have two load balancers. Each load balancer dispatches traffic to the healthy nodes of all the AZs. This means, in case of the loss of one AZ where one of the two load balancers is located, the other load balancer will still be available and will dispatch traffic to the healthy nodes, and their healthy `ingress-nginx` pods, on the other AZs.
```